<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Bronze Age Echoes in Northeast Africa | Ludovica Molinaro</title> <meta name="author" content="Ludovica Molinaro"> <meta name="description" content="Global and Local Ancestry Deconvolution applied to human whole genome sequences from Northeast Africa"> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/snail_icon.jpg?02faf225628114575a8ccf18f41aa5d7"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://lm-ut.github.io/projects/NortheastAfrica/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Ludovica </span>Molinaro</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching</a> </li> <li class="nav-item "> <a class="nav-link" href="/CommSci/">CommSci</a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/publications/">publications</a> <div class="dropdown-divider"></div> <a class="dropdown-item" href="/projects/">projects</a> <div class="dropdown-divider"></div> <a class="dropdown-item" href="/teaching/">teaching</a> </div> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">Bronze Age Echoes in Northeast Africa</h1> <p class="post-description">Global and Local Ancestry Deconvolution applied to human whole genome sequences from Northeast Africa</p> </header> <article> <p>Supervised by Prof. Pagani (University of Padua + University of Tartu)</p> <p><b>Why is this project cool?</b></p> <style>.highlight{border:2px solid gray;padding:5px;margin-bottom:20px;display:inline-block}</style> <div id="jsonParagraph"></div> <script>const data={paragraph:"Our study delves into the genetic composition of Ethiopians, confirming that up to 50% of their DNA can be traced to an Eurasian component, which could potentially be linked to the enigmatic Sea People. Thanks to Local Ancestry deconvolution, we were able to study the genetic features of the Eurasian component specifically. This research sheds light on the intricate evolutionary history of Ethiopians and offers compelling insights into the historical migrations of this population. Our findings pave the way for further investigations into Ethiopia's genetic history, and highlight the importance of accounting for different ancestral components."},div=document.getElementById("jsonParagraph"),span=document.createElement("span");span.textContent=data.paragraph,span.classList.add("highlight"),div.appendChild(span);</script> <p>The presence in contemporary Ethiopians of genomic signatures of Eurasian origin has been reported by several authors and estimated to have arrived in the area from 3000 years ago. Several authors have tried to identify a plausible source population for such a signature, using haplotype-based methods on modern data (Pagani 2012) or single-site methods on modern (Pickrell, Hodgson) or ancient data (Lazaridis 2016). These studies did not reach a consensus on the plausible origin and spanned from an Anatolian or Sardinia-like proxy (Pickrell) to a broadly Levantine (Pagani) to a more specifically Levantine Neolithic component (Lazaridis). Given the ancient nature of the Eurasian migration to Ethiopia and the massive population movements and replacements that affected West Asia in the last 3000 years, a natural conclusion would be to rely on the inference made by using ancient data (aDNA) as proxy for the putative population sources. On the other hand, however, the deeply divergent, autochthonous African component which accounts for ~50% of most contemporary Ethiopian genomes was only modelled but not removed from the analyses relying on aDNA. We speculate that the presence of a highly divergent component may affect the overall allele frequency spectrum to an extent that would make it hard to control for it and, at once, to discern between subtly different, yet important, Eurasian sources (such as Anatolian or Levant Neolithic ones).</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/BronzeEchoes-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/BronzeEchoes-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/BronzeEchoes-1400.webp"></source> <img src="/assets/img/BronzeEchoes.jpg" class="img-fluid rounded z-depth-1" width="auto" height="auto" title="Ancestry Components in Northeast Africa" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Ancestry Components in Northeast Africa. </div> <p>To overcome these potential biases we here propose to re-assess pattern of allele sharing between the Eurasian component of Ethiopians (here called “NAF” for Non-African) and ancient and modern proxies from the Levant and the broader Mediterranean area after having extracted NAF from Ethiopians through ancestry deconvolution. As shown elsewhere (Yelmen et al. 2019) we could therefore i) exploit high quality modern data and ii) concentrate the power of allele sharing tools on genetic fractions with no or reduced African contributions.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/SummaryBronzeEchoes-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/SummaryBronzeEchoes-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/SummaryBronzeEchoes-1400.webp"></source> <img src="/assets/img/SummaryBronzeEchoes.jpg" class="img-fluid rounded z-depth-1" width="auto" height="auto" title=" Experimental Design and approach" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> Experimental Design and approach. </div> <p>Such an approach, while potentially beneficial, may introduce a novel source of bias linked with the performance of the ancestry deconvolution process, which we aim to explore. Particularly, after devonvolving the 120 analysed Ethiopian genomes (Pagani 2015) we assigned each genomic SNP into one of the following four categories based on the method likelihoods: 1) confidently non African (called NAF); 2) grey area, towards non Africa (called “X”); 3) grey area, towards Africa (called “Y”) and 4) confidently African (AF, consistently filtered out from our analyses). While basing our inference on the NAF component alone, we here demonstrate that the component X does account for a minority of the genome and, when analysed together with NAF does not change the quality of our conclusion.</p> <p>Furthermore, when joining together the NAF and AF confidently assigned components (to created “Joint” components) we recapitulate the signals of the untouched population, showing that the X and Y components are not holding a considerable or peculiar genetic signature and hence ruling out, in this study, the role of ancestry deconvolution as a potential source of artefacts.</p> <p>A preliminary exploration of the NAF genomes through projected PCA show them to fall among other Eurasian populations, indicating the quality of the deconvolution process, and highlighting an affinity with ancient populations with a high Anatolian Neolithic component (e.g. Anatolia_N and Minoans). Notably, several Jews populations from North Africa cluster with NAF as well. The affinity between Anatolian Neolithic and NAF was further highlighted by f3 outgroup statistics, which contrasted what was observed for the non-deconvoluted genomes. Overall, whole-genome sequences of all the Ethiopian populations appear closer to ancient Near Eastern populations such as: Natufian, Levant Neolithic, Minoans and Anatolian Neolithic. On the other hand, their NAF components appear closer to populations with a high Anatolian component instead of Levant (such as Minoans, Sardinians and Anatolia Neolithic).</p> <p>We followed the observed affinity between NAF and Anatolian Neolithic-like populations through a set of f4 tests aimed at refining through more and more stringent comparisons the best proxy population for the Eurasian layer. The whole-genomes, with both African and Non-African components, are significantly closer to a Levantine ancestry rather than Anatolic (Z-Score 2.98), with them being closer to Levant Chalcolithic individuals than Levant Neolithic. On the other hand, the NAF component is shown to be closer to a Neolithic ancestry from Anatolia rather than any Levantine one (Z-score -2.847). We further compare the best proxies for the Non African component using the top scoring populations from Outgroup f3 analyses. Minoans appear to be as close to NAF as Anatolian Neolithic individuals (Z-Scores &lt; 1). Given that our ability to pinpoint the actual source of the NAF component is inherently limited by the availability of ancient and modern populations, we used qpGraph and qpAdm to model NAF as a mixture of the major axes of genetic diversity that best describes the Mediterranean area at the time of the studied event, following Lazaridis et al. 2016. When looking at the untouched genomes, our qpAdm results replicate a Levant_N origin for the Eurasian component of Ethiopians (as in Lazaridis et al 2016). When looking at NAF, such an inference shifts towards a predominant Anatolian_N and CHG composition (~0.85 and ~0.15, respectively), remarkably similar to the modelling of Minoans and Tunisian Jews.</p> <div class="row"> <div class="col-sm mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/qpAdmBronzeEchoes-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/qpAdmBronzeEchoes-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/qpAdmBronzeEchoes-1400.webp"></source> <img src="/assets/img/qpAdmBronzeEchoes.jpg" class="img-fluid rounded z-depth-1" width="auto" height="auto" title="qpAdm on Ancestral Components" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> </div> <div class="caption"> qpAdm on Ancestral Components. </div> <p>In summary, from a genetic viewpoint, the Eurasian component detected in contemporary Ethiopians and believed to have arrived in the area from around 3 kya can be modelled as 85% Anatolian Neolithic and 15% CHG. While this result should not be interpreted as a connection between Neolithic Anatolia and Ethiopia, it offers a key to identify candidates among the available ancient and modern populations which, following geographic and chronological considerations, may be better proxies for the population or populations that brought the Eurasian component to East Africa. Of the ones analyzed here, Minoans seem to provide the two closest matches to NAF, adding on top of the genetic evidence a criteria of space/time compatibility. In conclusion, our work shows that when the mixing components are deeply differentiated such as in the case of contemporary Ethiopians, ancestry deconvolution increases the sensitivity of allele sharing tests and enables to fully exploit the uncomparably higher quality of modern genomes, which can act as reservoirs of ancient genetic components that would have otherwise been lost or overlooked.</p> </article> <h2>References</h2> <div class="publications"> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Molinaro2022" class="col-sm-8"> <div class="title">Ancestry deconvolution of Estonian, European and Worldwide genomic layers: a human population genomics excavation</div> <div class="author"> Ludovica Molinaro</div> <div class="periodical"> 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Nowadays, we often assume that our modern way of life is better than the past ones, and the more we go back in time, the more the past humans struggled with poor living conditions or unperforming technologies. It is no surprise that historical and archaeological studies tackle the curiosity of many of us: with them we are forced to reshape our modern belief system and gasp before how much past populations could in fact achieve. People in the past were organised, they moved, travelled, shared ideas, cultures and technologies: and we can still find traces of their encounters. This thesis aims to look at the residual traces of past encounters through the genetic data. The encounter of populations is referred to as admixture event and creates admixed populations. With an approach called ancestry deconvolution (AD) we can analyse the genetic mosaic of an admixed group, tracing the admixing contributors and further characterise the admixing event. Within the AD approaches, there are the methods referred to as Local Ancestry Inferences (LAI) that allows to infer which ancestry falls within a given inherited locus. Such a fine level of inferences come with constraints, highly limiting the LAI tools performances in some cases. In turn, while performing LA on an admixed population characterised by worldwide ancestries we can focus on discovering unknown past demographic events, applying LA on sub-continental admixtures, such as European populations, requires a methodological approach that must be designed to comprehend the LA limitations first. Eventually, we can leverage AD and LAI to ask questions beyond the realm of demography studies. It is possible to partially predict the probability of developing a certain phenotype with the Polygenic Scores (PS). However, the vast number of variables acting upon the genome are population-specific, and the inferences are hardly transferable from one population to another, hindering the application of PS to understudied or admixed populations. I present three studies that discuss the potentialities and limits of AD and LAI. I studied an admixture event where worldwide, highly divergent populations came together; the limitations in deconvoluting fine-scale admixture events, such as in European groups; and finally I applied the built knowledge to overcome the limitations in applying Polygenic Score on admixed populations.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="molinaro2019west" class="col-sm-8"> <div class="title">West Asian sources of the Eurasian component in Ethiopians: a reassessment</div> <div class="author"> Ludovica Molinaro, Francesco Montinaro, Burak Yelmen, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Davide Marnetto, Doron M Behar, Toomas Kivisild, Luca Pagani' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Scientific reports</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li></ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Evolutionary Biology Course 2018" class="col-sm-8"> <div class="title">Bronze Age echoes in modern human whole genome sequences from Northeast Africa</div> <div class="author"> Ludovica Molinaro</div> <div class="periodical"> 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Corso, Biologia Magistrale, Laurea Evoluzionistica, Biologia Dott, Relatore Pagani, Luca</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="molinaro2018human" class="col-sm-8"> <div class="title">Human evolutionary history of Eastern Africa</div> <div class="author"> Ludovica Molinaro, and Luca Pagani</div> <div class="periodical"> <em>Current Opinion in Genetics &amp; Development</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> </div> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Ludovica Molinaro. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?c9d9dd48933de3831b3ee5ec9c209cac" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>